chaos.py

```diff
@@ -20,15 +17,14 @@
 import github_api.comments
 
 # Has a sideeffect of creating private key if one doesn't exist already
-import encryption
-
-from github_api import exceptions as gh_exc
+# Currently imported just for the sideeffect (not currently being used)
+import encryption  # noqa: F401
```

Comment:
Need? No. Keep it for the side effect? Meh. I'm fine either way. The server  has the [public key available](http://chaosthebot.com/pubkey.txt) now.